tech/languages/csharp/mono-installation.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Mono
+page: csharp
+order: 1
+section: tech-languages
+description: C# provided by Mono: a Cross-platform, Open Source, .NET development framework
+---
+
+# Mono installation
+
+Fedora comes with [Mono](http://www.mono-project.com/) - a free and open source implementation of the .Net framework. To install it, simply type:
```

Comment:
I looked at https://github.com/developer-portal/content/blob/master/tech/languages/java/java-installation.md and that had a simple dash. I will use two dashes now...